docs/pages/_bundles/2CamelConfig.md

```diff
@@ -7,16 +7,84 @@ permalink: bundles/camelconfig
 # CamelConfig
 
 ## The goal of the bundle
-CamelConfig is a bundle designed to parse the Apache Camel xml flows and translate this into Symfony services.
+CamelConfig is a bundle designed to parse the Apache Camel XML flows and translate this **into Symfony services**.
 
 ## Main features
+
+* Easy to use and implement.
+* Route versioning.
+* Full support for complex routing, like *multicasts* or *recipient lists*.
+* Clear route definition, based **on Apache's specification**.
```

Comment:
In this case I would say based on "Apache camel"

Reply:
Fixed